sentry-ruby/lib/sentry/hub.rb

```diff
@@ -67,6 +67,23 @@ def pop_scope
       @stack.pop
     end
 
+    def get_transaction
+      current_scope.span
+    end
```

Comment:
We have both `get_span` and `get_transaction` on the scope. It could make sense to sometimes get the span to create a tree.
On the other hand. `get_transaction` should always return the root span (using the span recorder to return the first entry) see other SDKs how it's implemented there.